Gemfile

```diff
@@ -146,7 +146,10 @@ def override_gem(name, *args)
     raise "Trying to override unknown gem #{name}" unless (dependency = dependencies.find { |d| d.name == name })
     dependencies.delete(dependency)
 
-    gem name, *args
+    calling_file = caller_locations.detect {|loc| !loc.path.include?("lib/bundler")}.path
```

Comment:
Am I the only one who is totally 'fine' with not having spaces after `{` and before `}` ?  It feels like noise.  This line looks 'fine' to me.